$OpenBSD$

Fix memory allocation/deallocation for Python 2.5 and above
(from Debian's patchset for python-ogg)

--- src/pyoggpacket.c.orig	Mon Oct  7 01:04:57 2002
+++ src/pyoggpacket.c	Tue Jun  1 15:38:26 2010
@@ -60,7 +60,7 @@ py_ogg_packet_from_packet(ogg_packet *op)
 static void
 py_ogg_packet_dealloc(PyObject *self)
 {
-  PyMem_DEL(self);
+  PyObject_Del(self);
 }
 
 static PyObject*
